implementation/src/main/java/io/smallrye/config/Converters.java

```diff
@@ -406,7 +408,7 @@ public T convert(final String value) {
             if (pattern.matcher(value).matches()) {
                 return delegate.convert(value);
             }
-            throw new IllegalArgumentException("Value does not match pattern " + pattern + " (value was \"" + value + "\")");
+            throw new IllegalArgumentException("Value does not match pattern " + pattern + ORIGINAL_VAL_PREFIX + value + "\")");
```

Comment:
Maybe these should be updated to use `String.format` instead?

Reply:
Good point, I was being lazy